starlingcaptureapi/numbers.py

```diff
@@ -0,0 +1,69 @@
+from . import config
+from .file_util import FileUtil
+
+import logging
+import requests
+
+_logger = logging.getLogger(__name__)
+
+file_util = FileUtil()
+
+_COMMIT_URL = "https://node.numbersprotocol.io/version-test/api/1.1/wf/commit"
+
+
+def commit(asset, asset_tree, author):
+    """Register a commit to the integrity blockchain.
+
+    https://github.com/numbersprotocol/enterprise-service/wiki/Web-3.0-API#nit-commit
+
+    Args:
+        asset: path to the asset file
+        asset_tree: path to asset JSON
+        author: path to author JSON
+
+    Raises:
+        any file I/O errors
+        JSON decoding errors
+
+    Returns:
+        Transaction hash string. If registration fails it returns None.
```

Comment:
@anaulin do you think we should raise here when fail, or return None? What would be more consistent with the rest of the system?

Reply:
I choose to log errors and not raise anything to match the behaviour of `iscn.py`, which is the module most similar in functionality to this one. Happy to change if needed though.